### ISSUE TITLE
Allow uploading files (and other multipart/form-data) via drill

### ISSUE THREAD

**Issue by asonix** [fcsonline/drill#75]
It would be helpful for testing file-upload APIs

**Comment by fcsonline** [comment]
Indeed. I would be a really nice addition and it is relatively easy to implement.

**Comment by LetsMelon** [comment]
Is there a development branch for this feature that I can try out, as I would need file upload for my intended use?

**Comment by fcsonline** [comment]
Nope. It would be awesome to have but I have no time to work on this right now. If someone wants to try...
